cnxarchive/database.py

```diff
@@ -60,19 +64,33 @@ def _read_sql_file(name):
 
 
 def initdb(settings):
-    """Initialize the database from the given settings."""
-    connection_string = settings[config.CONNECTION_STRING]
+    """Initialize the database from the given settings.
+       If a superuser is given in the settings then the
+       function will use the superuser to initialize the
+       database.
+    """
+    if settings['super-connection-string']:
+        connection_string = settings['super-connection-string']
+    else:
+        connection_string = settings['db-connection-string']
     with psycopg2.connect(connection_string) as db_connection:
         with db_connection.cursor() as cursor:
             for schema_filepath in DB_SCHEMA_FILE_PATHS:
                 with open(schema_filepath, 'r') as f:
                     cursor.execute(f.read())
+            # allow standard user to access the  'plpythonu' langauge.
+            cursor.execute(
+                "ALTER LANGUAGE plpythonu OWNER TO {user}".format(**settings))
```

Comment:
That's actually the way that I tried to implement the SQL the first time.  The problem is that when you use

```python 
cursor.execute('ALTER LANGUAGE plpythonu OWNER TO %(user)s', settings)
```
the psycopg library interprets users to be a string rather than a postgres identifier.  For example, if in python you set ' settings['user'] = 'karen' then resulting SQL command using the psycopg library would be
```SQL
ALTER LANGUAGE plpythonu OWNER TO 'karen'
```
The actual command needs to be 
```SQL
ALTER LANGUAGE plpythonu OWNER TO karen
```
I tried looking through the psycopg library to see if there was a way to change this but I didn't see one.

It actually bothers me to switch back and forth between these two different ways of making a SQL commands.  If I keep running into this issue I might just override the cursor cursor to hide this formatting change.